src/Core/Models/Mail/OrganizationUserInvitedViewModel.cs

```diff
@@ -11,6 +11,7 @@ public class OrganizationUserInvitedViewModel : BaseMailModel
         public string OrganizationNameUrlEncoded { get; set; }
         public string Token { get; set; }
         public string ExpirationDate { get; set; }
+        public bool OrgCanSponsor { get; set; }
```

Comment:
I think we are leaning more towards `Organization` than `Org` these days

Reply:
Done!